brave/browser/component_updater/widevine_cdm_component_installer.cc

```diff
@@ -405,10 +457,11 @@ void WidevineCdmComponentInstallerPolicy::RegisterWidevineCdmWithBrave(
     std::unique_ptr<base::DictionaryValue> manifest) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
   const std::string codecs = GetCodecs(*manifest);
+  bool supports_persistent_license = GetPersistentLicenseSupport(*manifest);
 
   content::PepperPluginInfo plugin_info;
   if (!MakeWidevineCdmPluginInfo(cdm_version, cdm_install_dir, codecs,
-                                 &plugin_info)) {
+                                 supports_persistent_license, &plugin_info)) {
```

Comment:
This change (`supports_persistent_license`) should also apply to `AtomContentClient::AddContentDecryptionModules`

Reply:
Pushed a fixup.